prisonstaffhub-specs/src/test/groovy/uk/gov/justice/digital/hmpps/prisonstaffhub/specs/IepHistorySpecification.groovy

```diff
@@ -0,0 +1,86 @@
+package uk.gov.justice.digital.hmpps.prisonstaffhub.specs
+
+import org.junit.Rule
+import uk.gov.justice.digital.hmpps.prisonstaffhub.mockapis.Elite2Api
+import uk.gov.justice.digital.hmpps.prisonstaffhub.mockapis.OauthApi
+import uk.gov.justice.digital.hmpps.prisonstaffhub.model.TestFixture
+import uk.gov.justice.digital.hmpps.prisonstaffhub.pages.IepHistory
+
+import static uk.gov.justice.digital.hmpps.prisonstaffhub.model.UserAccount.ITAG_USER
+
+class IepHistorySpecification extends BrowserReportingSpec {
+
+    @Rule
+    Elite2Api elite2api = new Elite2Api()
+
+    @Rule
+    OauthApi oauthApi = new OauthApi()
+
+    TestFixture fixture = new TestFixture(browser, elite2api, oauthApi)
+
+    def "should present iep history"() {
+        def offenderDetails = [
+                bookingId: -3,
+                bookingNo: "A00113",
+                offenderNo: "A1234AC",
+                firstName: "NORMAN",
+                middleName: "JOHN",
+                lastName: "BATES",
+                agencyId: "LEI",
+                assignedLivingUnitId: -3,
+                activeFlag: true,
+                dateOfBirth: "1999-10-27"
+        ]
+
+        def agencyDetails = [
+                agencyId: "LEI",
+                description: "Leeds",
+                agencyType: "INST"
+        ]
+        def userDetails = [
+                userId: "ITAG_USER",
+                firstName: "Staff",
+                lastName: "Member"
+        ]
+
+        elite2api.stubOffenderDetails('A1234AC', offenderDetails)
+        elite2api.stubAgencyDetails('LEI', agencyDetails)
+        elite2api.stubUserDetails('ITAG_USER', userDetails)
+        elite2api.stubGetIepSummaryWithDetails('-3')
+
+        given: "I am logged in"
+        fixture.loginAs(ITAG_USER)
+
+        when: "I view the IEP history page"
+        go "/offenders/A1234AC/iep-history"
```

Comment:
I think to use the `url` variable in the `IepHistory` page you can do:

 `to IepHistory` instead of the `go`

According to [this](http://www.gebish.org/manual/current/#scripting-with-page-objects) it will also do the `at IepHistory` bit too.

Reply:
Thank you, have changed it to use that!